challenge_1/go/rchicoli/reverse.go

```diff
@@ -0,0 +1,10 @@
+package rchicoli
+
+func ReverseString(s string) string {
+	l := len(s) - 1
+	r := make([]byte, l+1)
```

Comment:
Go actually uses `rune` for its character type. This works for ASCII text but not for the full range of Unicode characters.